### PR TITLE
GitHub rrr

### PR DESCRIPTION
**Pull request by RamRajasagi**
adding me

### CODE REVIEW

File: _path/github.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: RamRajasagi
+latitude: 38.650729
+longitude: -77.288175
+
```
Comment (hectorsector) [inline _path/github.json]: @RamRajasagi remove this file altogether

File: _pins/GitHub-Student-RRR.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+githubHandle: GitHub-Student-RRR
+latitude: 38.650729
+longitude: -77.288175
```
Comment (hectorsector) [inline _pins/GitHub-Student-RRR.json]: @RamRajasagi just add `---` to line 5 and you should be set!

File: _path/github.json (hunk 1 above)
Comment (RamRajasagi) [reply _path/github.json]: @hectorsector Done TQ :)

File: _path/github.json [hunk 3]
```diff
@@ -0,0 +1 @@
...
```
Comment (hectorsector) [inline _path/github.json]: @RamRajasagi remove this file

**Pull request closed without merging**